### PR TITLE
Resolve "[Settings] Update UI to add Ledger THORChain"

### LINKED ISSUE #1653: [Settings] Update UI to add Ledger THORChain
Screenshot from 2021-08-06 10-07-20

^ https://www.figma.com/file/LgRqZCGMShZ2mJ702MIqyV/ASGARDEX-2020?node-id=1307%3A7822

^ That's very similar we already had for BNB and BTC few months ago ...

Pls disable "ADD LEDGER" button as long as we don't have finished all business logic for Ledger THORChain (business logic will be added with another PRs)

Part of #1570

### PR DESCRIPTION
**Pull request by gromxyz**
- add button for adding ledger
- show dummy ledger address

Closes #1653

### CODE REVIEW

File: src/renderer/components/wallet/settings/Settings.tsx [hunk 1]
```diff
@@ -93,16 +97,29 @@ export const Settings: React.FC<Props> = (props): JSX.Element => {
...
       </Styled.AddressContainer>
     ),
     [clickAddressLinkHandler, selectedNetwork]
   )
 
+  const renderAddLedger = useCallback(
+    () => (
+      <Styled.AddLedger>
+        <PlusIcon />
+        <div style={{ marginLeft: '5px', textTransform: 'uppercase' }}>
```
Comment (veado) [inline src/renderer/components/wallet/settings/Settings.tsx]: Extract styles to `Styled.*`

**Review by veado — CHANGES_REQUESTED** [review]
Pls share a screen shot.

File: src/renderer/components/wallet/settings/Settings.tsx [hunk 2]
```diff
@@ -120,6 +137,12 @@ export const Settings: React.FC<Props> = (props): JSX.Element => {
                       <Styled.ChainContent key={j}>
                         <Styled.AccountPlaceholder>{acc.name}</Styled.AccountPlaceholder>
                         {renderAddress(item.chainName, acc.address)}
+                        {item.chainName.toLowerCase() === 'thor' && (
```
Comment (veado) [inline src/renderer/components/wallet/settings/Settings.tsx]: Use `isThorChain` of `chainHelper.ts`

File: src/renderer/i18n/de/ledger.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { LedgerMessages } from '../types'
 
 const ledger: LedgerMessages = {
+  'ledger.add.device': 'Add ledger - DE',
```
Comment (veado) [inline src/renderer/i18n/de/ledger.ts]: `Ledger hinzufügen`

**Comment by gromxyz** [comment]
Screenshot 2021-08-10 at 18 39 46
Screenshot 2021-08-10 at 18 39 23

**Pull request merged**